guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -4,7 +4,7 @@
 Before installing {Project} in an IPv6 network, ensure that you meet the following requirements:
 
 ifdef::satellite[]
-* If you plan to provision hosts from {Project} or {SmartProxies}, you must install {Project} and {SmartProxies} on {RHEL} version 7.9 or higher because these versions include the latest version of the `grub2` package.
+* If you plan to provision hosts from {Project} or {SmartProxies}, you must install {Project} and {SmartProxies} on {RHEL} version 8.0 or higher because these versions include the latest version of the `grub2` package.
```

Comment:
I don't think this is right. It specifically needed a feature from grub2 that shipped in RHEL 7.9. If it's present in RHEL 8.0, then the whole line can be removed. I didn't check, could also be RHEL 8.1 or 8.2.

Reply:
> I don't think this is right. It specifically needed a feature from grub2 that shipped in RHEL 7.9. If it's present in RHEL 8.0, then the whole line can be removed. I didn't check, could also be RHEL 8.1 or 8.2.

Ok. I am trying to get accurate information. :+1:

Reply:
Hi @evgeni  @maximiliankolb  Do you have something for this?

Reply:
@mjivraja I think you can remove the whole ifdef, because as ekohl says, it is not needed for RHEL8.

Reply:
Alright @Lennonka :+1: 
Thanks

Reply:
Addressed :+1: